frontend/src/pages/ActivityReport/Pages/activitySummary.js

```diff
@@ -233,6 +237,38 @@ const ActivitySummary = ({
     </div>
   );
 
+  const validateCitations = () => {
+    const allGoals = [selectedGoals, goalForEditing].flat().filter((g) => g !== null);
```

Comment:
I wonder why it would be possible for a goal to be null?

Reply:
I think it has to do with the fact the these values are coming from the hook form and could not yet be set.

In the case of selected goals vs goal for editing, its dependent upon if they are actively editing a goal.